### PR TITLE
Remove unused flags

### PR DESCRIPTION
**Pull request by dongrie**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ CLI="ethereum-ics20-cli"
 ## erc20
 ```bash
 ${CLI} erc20 balance --rpc-address=${rpc_address} --wallet-address=${wallet_address} --denom=${denom}
-${CLI} erc20 transfer --rpc-address=${RPC_ADDRESS} --mnemonic="${MNEMONIC}" --ics20-bank-address=${ICS20_BANK_ADDRESS} --ics20-transfer-bank-address=${ICS20_TRANSFER_BANK_ADDRESS} --from-index=${from_index} --to-address=${to_address} --amount=${amount} --denom=${DENOM}
+${CLI} erc20 transfer --rpc-address=${RPC_ADDRESS} --mnemonic="${MNEMONIC}" --from-index=${from_index} --to-address=${to_address} --amount=${amount} --denom=${DENOM}
```
Comment (mattsu6666) [inline README.md:15]: I noticed that the README has fixed tag which is v0.0.3, but it is possible to forget to update. 
How about changing it as follows?

```
$ git clone https://github.com/datachainlab/ethereum-ics20-cli.git
$ cd ethereum-ics20-cli
$ go install
CLI="ethereum-ics20-cli"
```

Comment (dongrie) [reply README.md:14]: Indeed! Specifying a branch name is not a good idea. Removed
https://github.com/datachainlab/ethereum-ics20-cli/pull/8/commits/5bc770060368621074543255e01693c7825d3370

**Review by mattsu6666 — APPROVED** [review]
LGTM!

File: cmd/erc20/transfer.go [hunk 2]
```diff
@@ -49,8 +45,9 @@ func transferCmd() *cobra.Command {
 	return cmd
 }
 
-func transfer(ctx context.Context, rpcAddress string, mnemonic, ics20BankAddress, ics20TransferBankAddress string, fromIndex uint32, toAddress string, amount int64, denom string) error {
-	chain, err := geth.InitializeChain(ctx, rpcAddress, mnemonic, denom, ics20TransferBankAddress, ics20BankAddress)
+func transfer(ctx context.Context, rpcAddress string, mnemonic string, fromIndex uint32, toAddress string, amount int64, denom string) error {
+	dummyAddress := "0x0"
+	chain, err := geth.InitializeChain(ctx, rpcAddress, mnemonic, denom, dummyAddress, dummyAddress)
```
Comment (siburu) [inline cmd/erc20/transfer.go:50]: This is a bit hacky.
Why not remove contract accessors (e.g. ERC20Token) from Chain struct?
A contract accessor can be easily created if needed.

**Review by siburu — CHANGES_REQUESTED** [review]

File: cmd/erc20/transfer.go (hunk 2 above)
Comment (dongrie) [reply cmd/erc20/transfer.go]: https://github.com/datachainlab/ethereum-ics20-cli/pull/8/commits/4b1995816bcbb49437d726a7626938cb731c8393
Removed tokenAddress from NewChain and AddXXX func

Comment (siburu) [reply cmd/erc20/transfer.go]: Sorry for the confusion ... but I suggested that you should simplify the definition of Chain like below.
```
type Chain struct {
	chainID        int64
	mnemonicPhrase string
	keys           map[uint32]*ecdsa.PrivateKey
	Client        *client.ETHClient
}
```

Instantiation of `Erc20Token`, `ICS20Transfer` and `ICS20Bank` is very simple, and so it should be done at the location of use like below.
```
erc20Token, err := erc20.NewErc20(common.HexToAddress(tokenAddress), chain.Client)
if err != nil {
    <... error handling ...>
}
```

Comment (siburu) [reply cmd/erc20/transfer.go]: If I have overlooked anything, please let me know.

Comment (dongrie) [reply cmd/erc20/transfer.go]: Sorry, my understanding was wrong.
I have corrected it.
https://github.com/datachainlab/ethereum-ics20-cli/pull/8/commits/563115db80d27c9629c2eed36cc2cebd80efb231

**Review by siburu — APPROVED** [review]
Thank you for the good work!

**Pull request merged**